ci/kind/config-3nodes.yml

```diff
@@ -5,5 +5,11 @@ networking:
   podSubnet: 10.10.0.0/16
 nodes:
 - role: control-plane
+  extraPortMappings:
```

Comment:
I suppose this only add the mappings on control-plane node? I am wondering how we can guarantee that controller always being deployed on control-plane?